assets/training/vision/environments/automl-dnn-vision-gpu/environment.yaml

```
@@ -0,0 +1,11 @@
+image:
+  name: azureml/curated/automl-dnn-vision-gpu
+  os: linux
+  context:
+    dir: context
+    dockerfile: Dockerfile
+    template_files:
+    - Dockerfile
```

Please add conda_dependencies.yaml too:
```suggestion
    - Dockerfile
    - conda_dependencies.yaml
```